apps/server/src/modules/files-storage/service/files-storage-upload.service.spec.ts

```diff
@@ -97,27 +104,18 @@ describe('FilesStorageService upload methods', () => {
 			const { params, fileRecords, parentId: userId } = buildFileRecordsWithParams();
 
 			const file = createMock<FileDto>();
-			file.data = Readable.from('abc');
+			const readable = Readable.from('abc');
+			file.data = readable;
 			file.name = fileRecords[0].name;
-			file.mimeType = 'mimeType';
+			file.mimeType = 'dto-mime-type';
 
 			const fileSize = 3;
 
 			const fileRecord = createFileRecord(file.name, 0, file.mimeType, params, userId);
 			const { securityCheck, ...expectedFileRecord } = fileRecord;
 			expectedFileRecord.name = resolveFileNameDuplicates(fileRecord.name, fileRecords);
-
-			const getFileRecordsOfParentSpy = jest
-				.spyOn(service, 'getFileRecordsOfParent')
-				.mockResolvedValue([[fileRecord], 1]);
-
-			// The fileRecord._id must be set by fileRecordRepo.save. Otherwise createPath fails.
```

Comment:
Why did you remove the mock here and put it in each setup? Did it lead to problems?

Reply:
Mocks should be located in the setup function of the scenario